src/arithmetic/entity_funcs/entity_funcs.c

```diff
@@ -140,6 +168,16 @@ void Register_EntityFuncs() {
 	func_desc = AR_FuncDescNew("type", AR_TYPE, 1, 1, types, false);
 	AR_RegFunc(func_desc);
 
+	types = array_new(SIType, 1);
+	types = array_append(types, T_NULL | T_EDGE);
+	func_desc = AR_FuncDescNew("startNode", AR_STARTNODE, 1, 1, types, false);
```

Comment:
`AR_FuncDescNew` adds a new argument, `aggregate` which should be set to `false` for both functions